forc/src/ops/forc_clean.rs

```diff
@@ -31,12 +31,16 @@ pub fn clean(command: CleanCommand) -> Result<()> {
 
     // Run `cargo clean`, forwarding stdout and stderr (`cargo clean` doesn't appear to output
     // anything as of writing this).
-    process::Command::new("cargo")
-        .arg("clean")
-        .stderr(process::Stdio::inherit())
-        .stdout(process::Stdio::inherit())
-        .output()
-        .map_err(|e| e)?;
+    if find_manifest_dir(&this_dir, "Cargo.toml").is_some() {
+        process::Command::new("cargo")
+            .arg("clean")
+            .stderr(process::Stdio::inherit())
+            .stdout(process::Stdio::inherit())
+            .output()
+            .map_err(|e| e)?;
+    } else {
+        colour::yellow_ln!("WARNING: use of forc clean without a Cargo.toml file in ANY parent directory will result in a bypass of the cargo clean command.");
```

Comment:
I was just trying to find a way to let the user know what was happening, should we maybe just have it as a normal `println!` and use "NOTE:" instead of "WARNING:"?

Reply:
If we use `println_yellow_err` like in other places it will be of `Result` type, then I remembered `colour` just colors the `println` macro.

Reply:
I think we should use an existing function for now. If you think switching to `colour` is a good idea, we should open an issue to discuss why and then if we decide to go ahead with it, we should remove `termcolor` in favour of it in a dedicated PR.

We should avoid introducing multiple crates for doing the same thing as it can add to dependency bloat (`forc` already has a lot) and make things confusing for new contributors. For now, it's fine to `unwrap` or `expect` on the IO result produced by `println_yellow_err` to follow [how `std::println` behaves](https://doc.rust-lang.org/std/macro.println.html#panics). We should probably update `println_with_color` and `print_with_color` to `expect` the `Result` so that every use of the `println_<color>` doesn't require this, but that change is also better off with its own issue and PR.

Reply:
`termcolor` would definitely be better here, that's my bad!